cli/cmd/cluster_gcp.go

```diff
@@ -263,6 +265,15 @@ var _clusterGCPDownCmd = &cobra.Command{
 
 		fmt.Print("￮ spinning down the cluster ")
 
+		uninstallCmd := "/root/uninstall.sh"
+		if _flagClusterGCPDownKeepVolumes {
+			uninstallCmd += " --keep-volumes"
+		}
+		_, _, err = runGCPManagerAccessCommand(uninstallCmd, *accessConfig, nil, nil)
+		if err != nil {
+			exit.Error(err)
```

Comment:
For cluster down, it is generally preferable to have best-effort deletes. 

Consider the following scenario: the first `cluster-gcp down` request fails on the delete cluster step. In the next retry, the `cluster-gcp down` command fails at the removing the volumes step because the previous cluster down attempt prevented us from getting kubectl config. The user can't retry `cluster-gcp down` because it fails at an unrelated step.